handlers/update_resource.go

```diff
@@ -39,12 +43,25 @@ func (d *updateResourceEntry) Handle(params operations.UpdateResourceParams) mid
 		panic(err)
 	}
 
-	merged := d.mergeJSON(&existingResource.JSON, &newResource.JSON)
-	newResource = datautils.NewResource(merged).
-		WithExternalIdentifier(id).   // Don't allow changing druids
-		WithID(existingResource.ID()) // Ignore any passed in tacoIdentifier
+	err = d.verifyPayload(newResource, existingResource)
+	if err != nil {
+		return operations.NewUpdateResourceUnprocessableEntity()
```

Comment:
I would like to see a `WithPayload` here, so the user can see what they did wrong. Also add a test for this path

Reply:
I'm going to work through this one a bit more, but we may need to pair on it in the AM as it seems (at first pass) to be more complex than just sending the error to `WithPayload` - which I don't think should be a blocker here and would be happy to quickly follow up with to keep things smallish.